xml/OrganisationRegistrationAgency.xml

```diff
@@ -17,15 +17,17 @@
         is used to indicate Multilateral/International Agency, and XI which is
         used to indicate International Organisations. This follows ISO's
         convention of reserving codes starting with X for "private use".
+        
```

Comment:
I would suggest removing all of the content above and replacing with:

Organization identifier lists (formerly Organization Registration Agency). The values from this codelist are used to identify the particular list that an organization identifier was drawn from. The codelist provides a list of known identifier lists, including national company registers, NGO lists and international and multilateral lists - along with links to sources where specific organization numbers and identifiers can be looked up.